docs/getting-started/migrating-workflows-to-github-apps.rst

```diff
@@ -0,0 +1,112 @@
+Migrating Your Existing Workflows to Use GitHub Apps
+======================================================
+
+Dockstore 1.9.0 provides users with a way to keep their workflows automatically updated (instead of needing to manually refresh) by using GitHub apps.
+Here, we will go over how to migrate your existing Dockstore workflows to use GitHub apps. Also keep in mind that the migration process is done one branch/version at a time.
+This tutorial assumes that you have read both our overview of :doc:`GitHub apps </getting-started/github-apps>`. and the :doc:`GitHub apps section </getting-started/dockstore-workflows>`
+in workflow registration documentation, and are therefore familiar with the required  ``/.dockstore.yml`` file.
+
+GitHub App Installation
+-----------------------
+
+The first step to migrating a workflow is the same as adding a new workflow via GitHub apps: install our Dockstore GitHub app onto your repository or
+organization. You do this by navigating to ``/my-workflows``, clicking the ``+`` button on the left hand sidebar, selecting ``Register using GitHub Apps``, and then clicking
```

Comment:
Despite the redundancy, I'd go for adding a screenshot here regardless. At the very least, I think adding the link to the guide on the 'adding a new workflow via Github apps' text would be helpful.